### DISCUSSION TITLE
Offtopic - What are GitHub community "discussions"?

### DISCUSSION THREAD

**Discussion by Vadorequest** [vercel/next.js#10914] [Help] [answered]
What is this new "discussions" tab? I've never seen this github feature, it seems to be a beta feature, would you have any github official documentation about it? (I'm interested for other projects)

Also, is it meant to replace Spectrum community? Or are you just trying things out?

**Comment by timneutkens** [comment, marked as answer]
Hey! It's in beta currently and only enabled for Next.js.

See https://github.com/zeit/next.js/discussions/10436#discussioncomment-185 for more info!

**Comment by Vadorequest** [comment]
@simpsoka I didn't get how to mark a discussion as a "question", nor how to mark it resolved.

I believe it would be useful that the "Beta" tag would redirect to some guidelines regarding this feature
![image](https://user-images.githubusercontent.com/3807458/76347138-65f91e00-6306-11ea-84a3-7dd38e5ef307.png)

It'd help with basic questions, just like those 👆

**Reply by Timer** [reply]
When you created the discussion there should've been a checkbox—I've converted this into a question and marked it resolved.

**Comment by JesterOrNot** [comment]
So is there a way to enable this for other repos?

**Reply by Vadorequest** [reply]
Not until it's officially released I supposed, or you participate in the private beta.

**Reply by JesterOrNot** [reply]
Thanks for letting me know!
